gdal/gdal_merge.xml

```diff
@@ -0,0 +1,211 @@
+                    </param>
+                    <when value="no_ul_lr">
+                    </when>
+                    <when value="ul_lr">
+                        <param name="ulx" type="integer" label="The ulx extents of the output file" value="0" min="0"/>
+                        <param name="uly" type="integer" label="The uly extents of the output file" value="0" min="0"/>
+                        <param name="lrx" type="integer" label="The lrx extents of the output file" value="0" min="0"/>
+                        <param name="lry" type="integer" label="The lry extents of the output file" value="0" min="0"/>
+                    </when>
+                </conditional>        
+                <param name="v" type="boolean" truevalue="-v" falsevalue="" checked="false" argument="-v" label="Generate verbose output of mosaicing operations as they are done" />
+                <expand macro="gdal_param_separate"/>
+                <param name="pct" type="boolean" truevalue="-pct" falsevalue="" checked="false" argument="-pct" label="Grab a pseudo-color table from the first input image, and use it for the output. Merging pseudo-colored images this way assumes that all input files use the same color table" />
+                <param name="n" type="text" argument="-n nodata_value" label="Ignore pixels from files being merged in with this pixel value" value="nodata_value" />
+                <param name="a_nodata" type="text" argument="-a_nodata" label="Assign a specified nodata value to output bands" value="output_nodata_value" />
+                <param name="init" type="text" argument="-init values(s)" label="Pre-initialize the output image bands with these values. However, it is not marked as the nodata value in the output file. If only one value is given, the same value is used in all the bands" value="" />
+            </when>
+        </conditional>
+    </inputs>
+        
+    <outputs>
+        <data name="output" auto_format="true"/> <!--label="${input.name}.${of}-->
+    </outputs>
+    
+    <tests>
+        <test>
+            <param name="input" value="west.dem"/>
+            <param name="input" value="east.dem"/>
+            <param name="advanced" value="simple"/>
+            <output name="output" file="mergeddem.tif"/>
+        </test>
+        
+    </tests>
+    
+    <help><![CDATA[
+==========================
+Gdal_merge
+==========================
+**What it does**
+
+This utility will automatically mosaic a set of images. All the images must be in the same coordinate system and have a matching number of bands, but they may be overlapping, and at different resolutions. In areas of overlap, the last image will be copied over earlier ones.
+
+|
+
+**How to use it**
+
+Select as much raster files as you want from history to make the mosaic.
+
+Specifiy output format using short name (http://www.gdal.org/formats_list.html).
+
+|
+
+**Other options, advanced usage**
```

Comment:
@Alanamosse @yvanlebras what is your thinking about parameter dumps in help section. Different communities handle this differently. I personally tend to think that the description should be on the parameter level and not duplicated in the help section (DRY) and use the help-section for other more useful stuff.

Reply:
Some parameters documentation can be very consistent, maybe too big to be placed in parameter help section ? I tried to cut off the long ones to have a 'more readable' display. 
Put the complete documentation in the general help section can generate some duplication I agree.
Should we let the user look for complete doc through gdal's official website ? 
@yvanlebras

Reply:
Agree with you both. I think we can 1/ provide a hyperlink toward gdal website (now) and 2/ work to create a more human understable help... (in the next future if existing help is sufficient). This make sense ?

Reply:
:+1:

Reply:
Ok for me